tests/data/dataset_test.py

```diff
@@ -38,9 +38,9 @@ def test_as_arrays(self):
         dataset.index_instances(self.vocab)
         padding_lengths = dataset.get_padding_lengths()
         arrays = dataset.as_arrays(padding_lengths)
+        text1 = arrays["text1"]["words"]
```

Comment:
My issue was that I was reading through this test and wondered where "words" came from.  It's not obvious, and it's not the default.  I only know it's not the default because I made the default - someone who is just looking at the test might think that "words" is what you should normally use, because there's nothing obvious in this test that's changing something to "words".